src/test/java/bogdanSandu/tests/Tema5/Tests.java

```diff
@@ -0,0 +1,39 @@
+package bogdanSandu.tests.Tema5;
+
+import bogdanSandu.driver.BrowserManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Tests {
+    public static void main(String[] args) {
+        ChromeDriver driver = BrowserManager.createDriverAndGetPage();
```

Comment:
Please make separate methods for each page and call those methods from main. At the moment we have a test that does everything. That means that if something breaks, the following lines of code won't run. We'll consider separated methods like independent test cases. If one of the test cases fail, the other may still pass.